### PR TITLE
batches endpoint added 

### PR DESCRIPTION
**Pull request by amjadiftikhar**

### CODE REVIEW

File: src/server/api/controllers/batches.controller.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+const knex = require('../../config/db');
+// const Error = require('../lib/utils/http-error');
```
Comment (senner007) [inline src/server/api/controllers/batches.controller.js:2]: Remove the outcommented code

**Comment by MirSzabo** [comment]
Hi Amjad endpoint works as expected tested in postman 
good work!

but for some reason I can't see all the endpoints from swagger file on page http://localhost:5000/api/documentation/
I don't have this issue on develop branch so we should check it

**Comment by MirSzabo** [comment]
after update it looks ok 👍

**Comment by dpfernandes** [comment]
Approved

**Pull request merged**